src/nidigital/metadata/functions.py

```diff
@@ -2080,6 +2080,7 @@
     'IsSiteEnabled': {
         'documentation': {
             'description': 'TBD'
+            'note': 'The function returns an error if more than one site is specified.\n'
```

Comment:
Please include the updated file in `generated` folder in the review.

Reply:
I rebased onto master and included the generated files.